yaksh/views.py

```diff
@@ -2666,3 +2666,20 @@ def course_modules(request, course_id, msg=None):
     context = {"course": course, "learning_modules": learning_modules,
                "user": user, "msg": msg}
     return my_render_to_response('yaksh/course_modules.html', context)
+
+
+@login_required
+@email_verified
+def course_status(request, course_id):
+    user = request.user
```

Comment:
Instead of the export to CSV through js, I suggest it would be good if another argument, a Boolean one, is passed. If the Boolean is false it will render in HTML or else it will create a CSV file and dump the data.

Reply:
I felt it would be better to do because we can avoid request and unnecessary db query which would result in same data fetch which I am already fetching.

Reply:
Yes, but my way does the same thing, without the intereference of js

Reply:
I feel it is okay for now